recipe/check_imports.py

```diff
@@ -0,0 +1,89 @@
+print("import: 'PySide6.QtAsyncio'")
+import PySide6.QtAsyncio
+
+print("import: 'PySide6.QtConcurrent'")
+import PySide6.QtConcurrent
+
+print("import: 'PySide6.QtCore'")
+import PySide6.QtCore
+
+print("import: 'PySide6.QtDBus'")
+import PySide6.QtDBus
+
+print("import: 'PySide6.QtDesigner'")
+import PySide6.QtDesigner
+
+print("import: 'PySide6.QtGui'")
+import PySide6.QtGui
+
+print("import: 'PySide6.QtHelp'")
+import PySide6.QtHelp
+
+print("import: 'PySide6.QtNetwork'")
+import PySide6.QtNetwork
+
+print("import: 'PySide6.QtOpenGL'")
+import PySide6.QtOpenGL
+
+print("import: 'PySide6.QtOpenGLWidgets'")
+import PySide6.QtOpenGLWidgets
+
+print("import: 'PySide6.QtPdf'")
+import PySide6.QtPdf
+
+print("import: 'PySide6.QtPdfWidgets'")
+import PySide6.QtPdfWidgets
+
+print("import: 'PySide6.QtPrintSupport'")
+import PySide6.QtPrintSupport
+
+print("import: 'PySide6.QtQml'")
+import PySide6.QtQml
+
+print("import: 'PySide6.QtQuick'")
+import PySide6.QtQuick
+
+print("import: 'PySide6.QtQuickControls2'")
+import PySide6.QtQuickControls2
+
+print("import: 'PySide6.QtQuickTest'")
+import PySide6.QtQuickTest
+
+print("import: 'PySide6.QtQuickWidgets'")
+import PySide6.QtQuickWidgets
+
+print("import: 'PySide6.QtSql'")
+import PySide6.QtSql
+
+print("import: 'PySide6.QtSvg'")
+import PySide6.QtSvg
+
+print("import: 'PySide6.QtSvgWidgets'")
+import PySide6.QtSvgWidgets
+
+print("import: 'PySide6.QtTest'")
+import PySide6.QtTest
+
+print("import: 'PySide6.QtUiTools'")
+import PySide6.QtUiTools
+
+print("import: 'PySide6.QtWebChannel'")
+import PySide6.QtWebChannel
+
+print("import: 'PySide6.QtWebEngineCore'")
+import PySide6.QtWebEngineCore
+
+print("import: 'PySide6.QtWebEngineQuick'")
+import PySide6.QtWebEngineQuick
+
+print("import: 'PySide6.QtWebEngineWidgets'")
+import PySide6.QtWebEngineWidgets
+
+print("import: 'PySide6.QtWebSockets'")
+import PySide6.QtWebSockets
+
+print("import: 'PySide6.QtWidgets'")
+import PySide6.QtWidgets
+
+print("import: 'PySide6.QtXml'")
+import PySide6.QtXml
```

Comment:
This is a good idea however if I could increase the complexity slightly so it will explicitly quit & hopefully be a touch neater if we need to add to it

```suggestion
import importlib
import sys

MODULES = [
    'QtAsyncio', 'QtConcurrent', 'QtCore', 'QtDBus', 'QtDesigner', 'QtGui',
    'QtHelp', 'QtNetwork', 'QtOpenGL', 'QtOpenGLWidgets', 'QtPdf', 'QtPdfWidgets',
    'QtPrintSupport', 'QtQml', 'QtQuick', 'QtQuickControls2', 'QtQuickTest',
    'QtQuickWidgets', 'QtSql', 'QtSvg', 'QtSvgWidgets', 'QtTest', 'QtUiTools',
    'QtWebChannel', 'QtWebEngineCore', 'QtWebEngineQuick', 'QtWebEngineWidgets',
    'QtWebSockets', 'QtWidgets', 'QtXml'
]

failed_imports = []
for module in MODULES:
    full_module = f'PySide6.{module}'
    try:
        print(f"Importing {full_module}")
        importlib.import_module(full_module)
    except ImportError as e:
        failed_imports.append((full_module, str(e)))

if failed_imports:
    print("\nFailed imports:")
    for module, error in failed_imports:
        print(f"- {module}: {error}")
    sys.exit(1)

print("\nAll imports successful!")
```

Reply:
I considered doing it in a loop but the generated files conda build makes list them 1-by-1 so I attempted to replicate that as much as possible for consistency.

Reply:
Yeah I'm happy with that reasoning, no worries 👍